### PR TITLE
Fix performer/bio URL for TheNude scraper

### PR DESCRIPTION
**Pull request by Teda1**

### CODE REVIEW

File: scrapers/TheNude.yml [hunk 1]
```diff
@@ -35,13 +36,13 @@ xPathScrapers:
...
               - regex: Caucasian
-                with: "white"
+                with: "Caucasian"
               - regex: Black
-                with: "black"
-              - regex: Latin
-                with: "hispanic"
+                with: "Black"
+              - regex: Latina
+                with: "Latin"
```
Comment (bnkai) [inline scrapers/TheNude.yml:45]: The regex replace's where meant to keep the same ethnicity as the first performer scraper we had (freeones)
Some of the changes dont even change anything... I would either leave them as they were or remove all the postProcess

Comment (Teda1) [reply scrapers/TheNude.yml:45]: No problem. I forget I made other changes to this file locally I didn't mean to push.

File: scrapers/TheNude.yml [hunk 2]
```diff
@@ -23,7 +23,8 @@ xPathScrapers:
   performerScraper:
     performer:
       Name: (//meta[@itemprop="name"])[1]/@content
-      URL: (//meta[@itemprop="url"])[1]/@content
+      URL:
+        selector: //meta[@itemprop="url"]/@content
```
Comment (bnkai) [inline scrapers/TheNude.yml:27]: was there an issue with the previous selector?
this now fetches more than one results (you can check the stash log (set to debug) when scraping)

File: scrapers/TheNude.yml [hunk 3]
```diff
@@ -78,7 +79,13 @@ xPathScrapers:
       Piercings: 
         selector: //li/span[@class="list-quest"][contains(text(),'Piercings')]/../text()
       Image:
-        selector: (//meta[@itemprop="image"])[1]/@content
+        selector: //meta[@itemprop="url"]/@content
```
Comment (bnkai) [inline scrapers/TheNude.yml:82]: the selector as mentioned above gets 2 results
using the `(//meta[@itemprop="url"]/@content)[1]` instead should get us only the first

File: scrapers/TheNude.yml (hunk 2 above)
Comment (Teda1) [reply scrapers/TheNude.yml:27]: On my end, the previous selector was never populating URL with this scrapper. My stash instance runs in a docker container in Unraid. Changing the XPath to your suggestion led the URL to not be provided again. What behavior are you seeing on your end with the old XPath and my change?

File: scrapers/TheNude.yml (hunk 3 above)
Comment (Teda1) [reply scrapers/TheNude.yml:82]: This suggestion here also breaks the image selection.

Comment (bnkai) [reply scrapers/TheNude.yml:82]: damn.... idk why it happens. for me it seems to work but not all of the times...

**Pull request merged**